packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -749,3 +820,46 @@ export function normalizeHeaders(
 	}
 	return normalized;
 }
+
+type EmscriptenFS = any;
+
+/**
+ * Copies the MEMFS directory structure from one FS in another FS.
+ * Non-MEMFS nodes are ignored.
+ */
+function recreateMemFS(newFS: EmscriptenFS, oldFS: EmscriptenFS, path: string) {
```

Comment:
For posterity – it would be so, so lovely to directly reuse the `oldFS` object without going through the copying here.